### PR TITLE
Update dg

### PR DESCRIPTION
**Pull request by chashaobao123**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -890,7 +863,7 @@ Given below are instructions to test the app manually.
 2. Exporting a contact list with zero contacts
    1. Prerequisites: There are no people in DorManagerPro
    2. Test case: `export` <br>
-      Expected: A json file is exported to the data folder of the application, with the name of the json file being the time of export. The contents of the json file is as follows: { "persons" : [ ] }
+      Expected: A json file is exported to the data folder of the application, with the name of the json file being the time of export. The contents of the json file is as follows: { "persons" : \[ \] }
```
Comment (chashaobao123) [inline docs/DeveloperGuide.md:866]: this is the error that prof highlighted, should be fixed like this right?

Comment (SeanFoongjt) [inline docs/DeveloperGuide.md:866]: Should the entire thing be surrounded by backticks i.e. `{ "persons" : [] }`

**Review by SeanFoongjt — COMMENTED** [review]
Question on the placement of backticks

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -890,7 +863,7 @@ Given below are instructions to test the app manually.
 2. Exporting a contact list with zero contacts
    1. Prerequisites: There are no people in DorManagerPro
    2. Test case: `export` <br>
-      Expected: A json file is exported to the data folder of the application, with the name of the json file being the time of export. The contents of the json file is as follows: { "persons" : [ ] }
+      Expected: A json file is exported to the data folder of the application, with the name of the json file being the time of export. The contents of the json file is as follows: `{ "persons" : [ ] }`
```
Comment (chashaobao123) [inline docs/DeveloperGuide.md:866]: ok backticks added

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -205,118 +201,63 @@ The `edit` command is undoable.
...
+Step 3. The `DeleteCommand` is executed by the LogicManager and delete popup get displayed.
 
-The following activity diagram summarizes what happens when a user executes a new command:
+Step 4. If user click `Ok` on the pop-up, model updates the filteredPersonList and removes the contact, otherwise cancel the deletion.
 
-<puml src="diagrams/CommitActivityDiagram.puml" height="600" width="600"/>
+The following sequence diagram shows how a `delete` command goes through the `Logic` component:
 
-#### Design considerations:
+<puml src="diagrams/DeleteSequenceDiagram.puml" alt="Interactions inside the Logic Component for the `delete 1` Command" />
```
Comment (lancehaha) [inline docs/DeveloperGuide.md:242]: should we have standardize size for the puml diagram

**Review by SeanFoongjt — APPROVED** [review]
LGTM!

**Pull request merged**